### PR TITLE
Backend: Comment Implementation (merge after #29)

### PR DESCRIPTION
**Pull request by iggyray**
Changes:
Backend:

- Created `Comment` domain
- Created `Comment Repo` with `addComment` and `getComment` functions
- Created `/comments` endpoint

### CODE REVIEW

File: backend/main.go [hunk 1]
```diff
@@ -94,6 +94,8 @@ func run(ctx context.Context) int {
 	l.POST("/items", h.AddItem)
 	l.POST("/sell", h.Sell)
 	l.POST("/purchase/:itemID", h.Purchase)
+	l.GET("/:itemID/comments", h.GetCommentsByItemId)
+	l.POST("/:itemID/comments", h.AddComment)
```
Comment (tonklable) [inline backend/main.go:98]: I think they should be `/items/:itemID/comments` like `/items/:itemID/image`

Comment (iggyray) [reply backend/main.go]: @tonklable ok! changed!

**Comment by tonklable** [comment]
Can you rebase the branch to current main branch first?

**Comment by iggyray** [comment]
> Can you rebase the branch to current main branch first?

rebased!

**Comment by tonklable** [comment]
Something appears on my screen, so I committed again. Please check. If everything is fine, you can merge. @iggyray

**Pull request merged**